src/test/arena/arena_gravity.test.ts

```diff
@@ -26,12 +27,13 @@ describe("Arena - Gravity", () => {
 
   beforeEach(() => {
     game = new GameManager(phaserGame);
-    game.override.battleType("single");
-    game.override.moveset([Moves.TACKLE, Moves.GRAVITY, Moves.FISSURE]);
-    game.override.ability(Abilities.UNNERVE);
-    game.override.enemyAbility(Abilities.BALL_FETCH);
-    game.override.enemySpecies(Species.SHUCKLE);
-    game.override.enemyMoveset(new Array(4).fill(Moves.SPLASH));
+    game.override
+      .battleType("single")
+      .moveset([Moves.TACKLE, Moves.GRAVITY, Moves.FISSURE])
+      .ability(Abilities.UNNERVE)
+      .enemyAbility(Abilities.BALL_FETCH)
+      .enemySpecies(Species.SHUCKLE)
+      .enemyMoveset(SPLASH_ONLY);
   });
 
```

Comment:
If it's not too much trouble, could you add this link in a comment?
```suggestion

// Reference: https://bulbapedia.bulbagarden.net/wiki/Gravity_(move)

```